packages/OverflowMenu/stories/examples/OverflowMenuExample.js

```diff
@@ -16,7 +16,7 @@ const OverflowMenuExample = () => (
   <OverflowMenu align="bottom">
     <OverflowMenu.Content data-your-custom-attribute="your-custom-attribute-value" />
     <OverflowMenu.Trigger
-      data-pka-anchor="overflow-menu__trigger"
+      id="overflow-menu__trigger"
```

Comment:
This was bad advice.  I see now that Cypress is configured so that `cy.findByTestId()` will look for the `data-pka-anchor` attribute.  
https://github.com/acl-services/paprika/blob/de413ae7b2c329bf6b098bf9757c003779a49c8d/cypress/support/index.js#L20

So it _does_ make sense to use `data-pka-anchor` here instead of `id`. And that you should revert your changes in [overflowMenu.cypress.js](https://github.com/acl-services/paprika/pull/1217/files#diff-538e4cda82acc833604036694719fdd2daeba9841c4f423ab29893bf71d7e3e1)